src/App.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
```

Comment:
I believe this import can also be removed since we are not focusing on css just yet